texlive-scripts/fmtutil.pl

```diff
@@ -804,8 +843,12 @@ sub rebuild_one_format {
     $ENV{'TEXPOOL'} = cwd() . $sep . ($texpool ? $texpool : "");
   }
 
-  # in mktexfmtMode we must redirect *all* output to stderr
-  $cmdline .= " >&2" if $mktexfmtMode;
+  if ($opts{'quiet'} || $USE_FORKMANAGER) {
+    $cmdline .= " >$nul 2>$nul";
```

Comment:
If we run in quiet mode, or under fork manager, we do NOT show the output of the format creation run.
This complete log is still in the log file.